docs/api.rst

```diff
@@ -13669,23 +13669,25 @@ Managing calendar reminders
   :statuscode 500: Internal rotki error.
 
 
-.. http:delete:: /api/(version)/calendar/reminders
```

Comment:
Are you deleting the calendar reminder api docs entry? Seems so from the github diff